spring-cloud-common-security-config/spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/CommonSecurityAutoConfiguration.java

```diff
@@ -21,14 +21,12 @@
 import org.springframework.boot.autoconfigure.security.oauth2.resource.servlet.OAuth2ResourceServerAutoConfiguration;
 import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Import;
 
 @Configuration(proxyBeanMethods = false)
 @AutoConfigureBefore({
 		SecurityAutoConfiguration.class,
 		ManagementWebSecurityAutoConfiguration.class,
 		OAuth2ClientAutoConfiguration.class,
 		OAuth2ResourceServerAutoConfiguration.class})
-@Import({IgnoreAllSecurityConfiguration.class, OAuthSecurityConfiguration.class})
```

Comment:
Good catch. Looking comment in `IgnoreAllSecurityConfiguration`
> The org.springframework.cloud.common.security.enabled=true property disables this configuration...

Gives a feeling that something got tangled and "shared" stuff didn't work as planned and yet another "enabled" option was added.

It may be better to remove this auto-config, see what breaks i.e. in ctr and then bring something back in a better form.

Reply:
Created https://github.com/spring-cloud/spring-cloud-dataflow/issues/5926 to track this